src-vue/src/views/DeveloperView.vue

```diff
@@ -53,6 +61,30 @@ export default defineComponent({
                 type: 'success',
                 position: 'bottom-right'
             });
+        },
+        async disableAllModsButCore() {
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+            await invoke("disable_all_but_core_caller", { gameInstall: game_install }).then((message) => {
+                console.log(message);
```

Comment:
This should be removed.

Reply:
Done with 3359b95eb3defa968ee391bc9ca7426754e5e78f